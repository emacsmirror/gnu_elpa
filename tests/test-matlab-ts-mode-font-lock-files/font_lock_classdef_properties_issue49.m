% -*- matlab-ts -*-
%
% See: https://github.com/acristoffers/tree-sitter-matlab/issues/49
% where p1 double wasn't being parsed correctly.

classdef font_lock_classdef_properties_issue49
    properties
        p0
        p1 double;
    end
end
