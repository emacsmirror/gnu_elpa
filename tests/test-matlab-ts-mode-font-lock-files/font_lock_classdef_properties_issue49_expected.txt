c ccc ccccccccc ccc
c
c cccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
c ccccc cc cccccc cccccc ccccc cccccc cccccccccc

kkkkkkkk fffffffffffffffffffffffffffffffffffff
    kkkkkkkkkk
        PP
        PP ttttttD
    kkk
kkk
